docs/mkdocs/docs/api/macros/index.md

```diff
@@ -29,6 +29,7 @@ header. See also the [macro overview page](../../features/macros.md).
 
 ## Type conversions
 
+- [**JSON_DISABLE_ENUM_SERIALIZATION**](json_disable_enum_serialization.md) - switch off default parse/serialization functions for enums
```

Comment:
I think "Serialization/deserialization macros" is a better fit, isn't it?

Reply:
I though so too, but have seen `parse` used more in the documentation. But it does makes sense, you parse json and then deserialize it into specified type.

Changed it so it is serialization/deserialization and reflected this change in `json_disable_enum_serialization.md`.